app.js

```diff
@@ -1,19 +1,26 @@
 const log = require('./app/lib/logger');
+const downloadFile = require('./app/lib/downloadFile');
 const removeColumns = require('./app/lib/removeColumns');
 const getLatestPeriod = require('./app/lib/getLatestPeriod');
 const removeOldRecords = require('./app/lib/removeOldRecords');
-const downloadFile = require('./app/lib/downloadFile');
+const convertToJson = require('./app/lib/convertToJson');
 
 function handleError(err) {
   log.error('Processing failed', err);
 }
 
-Promise
-  .resolve(log.time('Downloading and transforming POMI data took'))
-  .then(downloadFile)
-  .then(removeColumns)
-  .then(getLatestPeriod)
-  .then(removeOldRecords)
-  .then(() => log.timeEnd('Downloading and transforming POMI data took'))
-  .catch(handleError);
+function app() {
+  return Promise
+    .all([
+      log.info('Starting download and transformation of POMI data'),
+      log.time('Downloading and transforming POMI data took')])
+    .then(downloadFile)
```

Comment:
I rewrote it so it would be easy to comment out line 17 if the file didn't need to be downloaded again. It also has the benefit of everything being contained within a promise (including the `log.xxx()`) which I think more aesthetically pleasing